### DISCUSSION TITLE
lightgallery is not working

### DISCUSSION THREAD

**Discussion by Sp-cy** [uPagge/uBlogger#130] [General]
Hello!
I can't make the lightgallery work. I'm using the Image shortcode and it doesn't seem to work in the documentation page either. 

Here's a post that should have it enabled: 

https://blog.sp-cy.com/en/p/shakkablood-is-making-covers-of-covers/

here's the .md :

```md
---
title: "SHAKKABLOOD Is Making Covers of Covers"
subtitle: "Reinterpretations of electronic music classics."
date: 2021-04-01T17:07:22-06:00
draft: false
author: "Spicy"
description: "Reinterpretations of electronic music classics."
url: "/en/p/shakkablood-is-making-covers-of-covers/"  
page:
    theme: "wide"
upd: ""
authorComment: ""

tags: ["3D", "Visual Design", "EDM", "Electronic Music"]
categories: ["Art"]

hiddenFromHomePage: false
hiddenFromSearch: false

resources:
- name: "featured-image"
  src: "featured-image.jpg"
- name: featured-image-preview
  src: featured-image-preview.jpg

featuredImage: ""
featuredImagePreview: ""
images: ["SHAKKABLOOD-KILLTHENOIS-OCCULTCLASSIC-02.jpg"]

lightgallery: true

toc:
  enable: false

math:
  enable: false

license: ""
---

<!--more-->
Thanks to his 3D reinterpretations of Kill The Noise's Occult Classic cover I discovered SHAKKABLOOD's work and it's so incredible I wanted to share it here. 

The color palettes, the lightning and textures in his work pieces are all in perfect harmony making it amazing to witness. 

{{< image src="SHAKKABLOOD-KILLTHENOIS-OCCULTCLASSIC-01.jpg" >}}
{{< image src="SHAKKABLOOD-KILLTHENOIS-OCCULTCLASSIC-02.jpg" >}}
{{< image src="SHAKKABLOOD-KILLTHENOIS-OCCULTCLASSIC-03.jpg" >}}

Via [Twitter](https://twitter.com/shakkablood/status/1377360627518201859)

I clicked in a bit more and found his reinterpretation of Skrillex - Bangarang.

{{< image src="SHAKKABLOOD-SKRILLEX-BANGARANG-01.jpg" >}}
{{< image src="SHAKKABLOOD-SKRILLEX-BANGARANG-02.jpg" >}}
{{< image src="SHAKKABLOOD-SKRILLEX-BANGARANG-03.jpg" >}}

Via [Twitter](https://twitter.com/shakkablood/status/1377266153203105794)


Look up for SHAKKABLOOD's work:
 - [Instagram](https://www.instagram.com/shakkablood/)
 - [Twitter](https://twitter.com/shakkablood)
 - [Web Page](https://artinghokasiyan.com/)
```

**Comment by uPagge** [comment]
if i'm not mistaken the galleries have been removed to facilitate the basic version of the theme.  you can use any library for galleries if you need them

**Reply by Sp-cy** [reply]
I would have added it if I knew how to do that. I'm not a developer, I learned HUGO just to have my own website. This theme said it had a gallery and that was one of the reasons I changed to this theme. I do also love the schema breadcrumbs and the general style but I might have to change theme for something that works better out of the box for my needs. Thank you for your answer, for the effort and time you've put into this theme. I guess I will have to learn how to do all this eventually to get the things I would like. Feel free to delete this discussion thread. Much respect, uPagge. Thanks again.

**Reply by uPagge** [reply]
try creating a discussion thread on the hugo forum thread.  I think there will be people there who will help you add some gallery, since it is not particularly difficult to do
